mw-webapp/src/model/DTOPartialModel/WayPartialDTO.ts

```diff
@@ -0,0 +1,90 @@
+import {timestampType} from "fireschema";
+import {z} from "zod";
+
+export const WAY_UUID_FIELD = "uuid";
+export const WAY_NAME_FIELD = "name";
+export const WAY_OWNER_UUID_FIELD = "ownerUuid";
+export const WAY_MENTOR_UUIDS_FIELD = "mentorUuids";
+export const WAY_CREATED_AT_FIELD = "createdAt";
+
+export const WayPartialDTOSchema = z.object({
```

Comment:
we don't need to hav such such schema, make a generic function which makes any zod scema optional

Reply:
Used .partial method